Assets/Scripts/Characters/Human/Hero.cs

```diff
@@ -84,7 +84,8 @@ public class Hero : Human
     private float flareTotalCD = 30f;
     private Transform forearmL;
     private Transform forearmR;
-    private float gravity = 20f;
+    private float Gravity => 20f * gravityModifier;
+    private float gravityModifier = GameSettings.Global.Gravity ?? 1;
```

Comment:
Ticket said that the Gravity setting had to be nullable, would it be an idea to just give it a default value of 1 so we don't end up having to do null checks like this every time we want to read it?

Reply:
wait u can make a custom map and have gravity like the moon? that's poggers